src/controllers/update/update.filing.date.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { NextFunction, Request, Response } from "express";
+import { Session } from "@companieshouse/node-session-handler";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { isActiveFeature } from "../../utils/feature.flag";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { postTransaction } from "../../service/transaction.service";
+import { createOverseasEntity, updateOverseasEntity } from "../../service/overseas.entities.service";
+import { OverseasEntityKey, Transactionkey } from '../../model/data.types.model';
+import { ApplicationData } from "../../model/application.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    return res.render(config.UPDATE_FILING_DATE_PAGE, {
+      backLinkUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
+      templateName: config.UPDATE_FILING_DATE_PAGE
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = async(req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const session = req.session as Session;
+
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022)) {
```

Comment:
If we don't have the web session (in future) then don't we need to capture the presenter details before? So this may be too late to start the save/resume. Need to see a design for Save&Resume to review this.

Reply:
@AndreKramer7654 , the presenter details are captured after this screen

Reply:
I've queried this with product as it seems a little strange to ask for presenter details after asking presenter to enter date.